src/components/decimal/decimal.component.tsx

```diff
@@ -179,6 +158,29 @@ export const Decimal = React.forwardRef(
       [getSeparator, isNaN, l, locale, precision]
     );
 
+    const emptyValue = allowEmptyValue
+      ? ""
+      : formatValue(`${(0).toFixed(precision)}`);
```

Comment:
```suggestion
      : formatValue((0).toFixed(precision));
```

`toFixed` already returns a string so there's absolutely no need to wrap it in backticks and `${...}`, which for a single value does nothing other than convert it to a string 😉

Reply:
Awh I hadn't realised that. I'll get them removed then 👍🏻